content/blog/2022/2022-08-02-performance-with-completablefutures.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "How to use CompletableFuture for better performance during API calls"
+categories: ["Java"]
+date: 2022-02-08T11:00:00 
+authors: [Bledi]
```

Comment:
Please add yourself to the file `data/authors.yml` and ideally also upload a profile picture because I'm planning to create author pages at some point.